### PR TITLE
Add support for IPv6 to No-IP Dynamic DNS. Feature #10256

### PR DESCRIPTION
**Pull request by csobankesmarki**
- [ ] Redmine Issue: https://redmine.pfsense.org/issues/10256
- [ ] Ready for review

### CODE REVIEW

File: src/etc/inc/dyndns.class [hunk 1]
```diff
@@ -507,6 +511,29 @@
...
+					if ($this->_dnsPort) {
+						$port = ":" . $this->_dnsPort;
+					}
+					if (($this->_dnsService == "noip-free-v6") &&
+					    ($this->_forceUpdateNeeded == true) &&
+					    ($this->_dnsDummyUpdateDone == false)) {
+						// Update the IP to a dummy value to force No-IP free accounts to see a change.
+						$iptoset = "fd00:d::1";
+						$this->_dnsDummyUpdateDone = true;
+						log_error(sprintf(gettext('Dynamic DNS %1$s (%2$s): Processing dummy update on No-IP free account. IP temporarily set to %3$s'), $this->_dnsService, $this->_dnsHost, $iptoset));
```
Comment (rbgarga) [inline src/etc/inc/dyndns.class]: Please break this long line improve readability

File: src/etc/inc/dyndns.class [hunk 2]
```diff
@@ -507,6 +511,29 @@
...
+					} else {
+						$iptoset = $this->_dnsIP;
+					}
+					curl_setopt($ch, CURLOPT_URL, $server . $port . '?username=' . urlencode($this->_dnsUser) . '&pass=' . urlencode($this->_dnsPass) . '&h[]=' . $this->_dnsHost.'&ipv6=' . $iptoset);
```
Comment (rbgarga) [inline src/etc/inc/dyndns.class]: Please break this long line improve readability

File: src/etc/inc/dyndns.class (hunk 1 above)
Comment (csobankesmarki) [reply src/etc/inc/dyndns.class]: Adopted suggestion: broke the line.

File: src/etc/inc/dyndns.class (hunk 2 above)
Comment (csobankesmarki) [reply src/etc/inc/dyndns.class]: Adopted suggestion: broke the line.

**Pull request merged**